pym/bob/input.py

```diff
@@ -2550,12 +2551,20 @@ def __init__(self):
             schema.Optional('key') : str,
             schema.Optional('sasToken"') : str,
         })
+        artifactoryArchive = baseArchive.copy()
+        artifactoryArchive.update({
+            'url' : str,
+            schema.Optional('key') : str,
+            schema.Optional('properties') : schema.Schema({ schema.Regex(r'^[A-Za-z_][A-Za-z0-9_]*$') : str }),
```

Comment:
Hmm, better leave the `properties` out. The use case is not really clear and it feels wrong to add such a feature just for Artifactory.